### PR TITLE
Dynamically Loading RequestsInbox based on Role [KIR-347]

### PR DESCRIPTION
**Pull request by ghost**
Before, we just had a mock email and just called the `approver/requests` API call. Now that we are aware of user email and user role, I want to dynamically load in the requests inbox based off your role.

- If you are a governor, load in governance requests (/governance/requests)
- Otherwise, you are an approver (because the Requests Inbox will not be accessible otherwise if you are just a plain user)

- **NOTE**: To test the inbox for approvers, you must:

1.  go through OneLogin to get your SAMLResponse token and paste it in as a query parameter to populate the email in the store. (DM me on slack if this isn't clear)
2. go to `/Reducers/initialState.js` and set `approver: false` since authorization isn't implemented.

### CODE REVIEW

File: src/State/helpers.js [hunk 1]
```diff
@@ -1,17 +1,34 @@
...
-    return {
-      ...request,
-      databasename: request.databasename || '', // placeholder for name property
-      requestdata: reqData,
-      description: reqData.description || '',
-      requeststatus:
-        request.requeststatus.charAt(0).toUpperCase() +
-        request.requeststatus.slice(1)
-    };
+    switch (role) {
```
Comment (robertwollny-pnmac) [inline src/State/helpers.js]: This was the only thing I can see getting an error when clicking on the request sent listItem.
Screen Shot 2019-12-06 at 11 22 10 AM

**Review by robertwollny-pnmac — COMMENTED** [review]
This is really good the redux store seems to all be functionally properly. And everything else looks good besides that one error.

File: src/State/helpers.js (hunk 1 above)
Comment (abbyjb) [reply src/State/helpers.js]: Trying to reproduce the error but I'm not running into it... can you tell me what you did to get here? @robertwollny-pnmac

Comment (ghost) [reply src/State/helpers.js]: fixed! try running it again on your machine

**Review by robertwollny-pnmac — APPROVED** [review]
It looks good to me all the issues I saw seem to be fixed and working as expected good job

**Review by Jake-Lindvall — APPROVED** [review]
Looks good to me too!

**Pull request merged**